DorisProject/src/app/pages/stock/add-stock/add-stock.component.ts

```diff
@@ -65,46 +76,86 @@ export class AddStockComponent implements OnInit {
   }
 
   openModal(request: string) {
-    const dialogRef = this.dialog.open(AddStockDialogComponent, {data: {textrequest: request}});
+    const dialogRef = this.dialog.open(ModalComponent, {
+      data: {textrequest: request, textresponse: this.responseModal, nameService:'Product'}});
```

Comment:
Codacy has a fix for the issue: [Replace `textrequest:·request,·textresponse:·this.responseModal,·nameService:'Product'}` with `⏎········textrequest:·request,⏎········textresponse:·this.responseModal,⏎········nameService:·'Product',⏎······},⏎····`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
      data: {
```